views/funding-overview/index.php

```diff
@@ -76,10 +156,177 @@
 </div>
 
 <style>
```

Comment:
So there's no way to put this block into it's own css file? it must be included with the view right?
It is heavily dependent on the Humhub styling so changing is always kind of hacky.. but it's ok

Reply:
I think, i've figure it out how to put styling in it's own css file.
We need to put resources (js/css/images) under **resources directory** under the module.
Then we define a php class named **Assets** under **assets directory**, there is also some other configs to do, check for example *tasks module* or *gallery module*.